src/components/LocationPage/ChartBlock.tsx

```diff
@@ -30,6 +34,11 @@ function ChartBlock(props: {
 }) {
   const { projections, metric, isMobile, region, stats } = props;
 
+  const metricToProvenanceMap: MetricToProvenance = makeMetricToProvenanceMap(
+    projections.primary.annotations,
+  );
+  const provenanceForMetric = metricToProvenanceMap[metric];
```

Comment:
I think at this point you could actually drop `MetricToProvenance`, `makeMetricToProvenanceMap()` and just call `getSourcesForMetric()` (which is otherwise unused I think?)...

Reply:
ahh yes, wow, tried a few different paths and didn’t delete the unused ones